src/components/arbitrum/SequencerFeed.tsx

```
@@ -0,0 +1,178 @@
+// import SequencerFeedMsg from './feeds/SequencerFeedMsg';
+// import SequencerFeedDetail from './feeds/SequencerFeedDetail';
+
+interface TableData {
+  id: number;
+  feedBlockHash: string;
+  l2BlockHash: string;
+  status: string;
+  timestamp: string;
+}
+
+interface SequencerMessage {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)